magstock/templates/static_views/legal_name.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
Agreed.  How we do this depends on what we're trying to do on a case-by-case basis, and this weekend I expect to have time to write some documentation/explanation of how I envision this happening in general for different types of per-event changes for different events.